__tests__/Itinerary.tests.js

```diff
@@ -0,0 +1,22 @@
+const Itinerary = require("../src/Itinerary");
+const Port = require("../src/Port");
+
+describe('Itinerary', () => {
+  let liverpool;
+  let newYork;
+  let itinerary;
+
+  beforeEach(() => {
+    liverpool = new Port('Liverpool');
+    newYork = new Port('New York'); 
+    itinerary = jest.fn();
+  });
```

Comment:
In this case, you are only using these values for one test so the `beforeEach()` block can be removed and the logic moved into the 'can have ports' test (example below)